services/auth/proxy/tests/unit/jwk-cache.test.ts

```diff
@@ -1,8 +1,8 @@
 import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest';
 import { _clearCachedJwks, getJwks, _returnCachedJwks } from '../../jwk-cache';
-import { AbortError } from 'node-fetch';
 import { logger } from '../../logger';
 import { logMessages } from '../../log-messages';
+import * as httpService from '../../../http/http-service';
```

Comment:
Can we use path aliases instead of relative paths? This should be akin to a package that we can use across `auth` and `chat` (... and more)